ops/services/postgres_db/nophi_user.tf

```diff
@@ -0,0 +1,40 @@
+locals {
+  schema_name = "public"
+
+  grant_command = <<EOF
+  psql \
+  --host ${azurerm_postgresql_server.db.fqdn} \
+  --username ${var.administrator_login}@${azurerm_postgresql_server.db.name} \
+  --dbname "${var.db_table}" \
```

Comment:
&lt;wince&gt; at that variable-naming self-own. 

Actually I'm confused—isn't the `@${NAME}` in the username an alternate spelling for the same value in `--dbname ${NAME}`? If so, are these two variables just different aliases for the same value?